geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AddIndexCommand.scala

```diff
@@ -0,0 +1,70 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import com.beust.jcommander.{JCommander, Parameter, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.hadoop.util.ToolRunner
+import org.locationtech.geomesa.jobs.index.AttributeIndexJob
+import org.locationtech.geomesa.tools.accumulo.GeoMesaConnectionParams
+import org.locationtech.geomesa.tools.accumulo.commands.AddIndexCommand.AddIndexParameters
+import org.locationtech.geomesa.tools.common.{AttributesParam, FeatureTypeNameParam}
+
+import scala.util.control.NonFatal
+
+class AddIndexCommand(parent: JCommander) extends CommandWithCatalog(parent) with LazyLogging {
+  override val command = "addindex"
```

Comment:
actually, could we rename to `add-attribute-index`? i think we're going to want a more general capability for managing any index later.